packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -563,13 +563,48 @@ export namespace EmrCreateCluster {
   }
 
   /**
-   * The launch specification for Spot instances in the instance fleet, which determines the defined duration and provisioning timeout behavior.
+   * Different allocation strategies that can be used by the EMR cluster
+   */
+  export class AllocationStrategy {
+    /** lowest-price allocation strategy. */
+    public static readonly LOWEST_PRICE = new AllocationStrategy('lowest-price');
+    /** capacity-optimized allocation strategy */
+    public static readonly CAPACITY_OPTIMIZED = new AllocationStrategy('capacity-optimized');
+
+    /**
+     * Create a new AllocationStrategy with an arbitrary allocation strategy.
+     *
+     * @param customAllocationStrategy the allocation strategy string,
+     *   for example "lowest-price"
+     */
```

Comment:
are there any docs links we can also point to?

Reply:
Added link to the Cfn docs.